baiji/serialization/test_yaml.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+from baiji.serialization import yaml
+
+class UnsafeToDump(object):
+    """Python classes should not by default be YAML dumpable in safe mode"""
+    pass
+
+unsafe_to_load = "!!python/name:baiji.serialization.test_yaml.UnsafeToDump ''\n"
+
+class TestYAML(unittest.TestCase):
+
+    def test_yaml_unsafe_dumps(self):
+        self.assertEqual(yaml.dumps(UnsafeToDump), unsafe_to_load)
```

Comment:
If you wanted, you could test the warning here via:
```py
with warnings.catch_warnings(record=True) as w:
    warnings.simplefilter("always") # Cause all warnings to always be triggered.
    self.assertEqual(yaml.dumps(UnsafeToDump), unsafe_to_load)
    self.assertEqual(len(w), 1)
    self.assertIn("Unsafe YAML serialization.", str(w[-1].message))
```
Not that you have to, but you could, if you wanted.